src/module/actor/sheet/item-summary-renderer.ts

```diff
@@ -96,14 +96,12 @@ export class ItemSummaryRendererPF2e<AType extends ActorPF2e> {
               })()
             : null;
 
-        const $priceLabel =
-            itemIsIdentifiedOrUserIsGM && item.system.stackGroup !== "coins"
-                ? ((): JQuery => {
-                      const price = item.price.value.toString();
-                      const priceLabel = game.i18n.format("PF2E.Item.Physical.PriceLabel", { price });
-                      return $(`<p>${priceLabel}</p>`);
-                  })()
-                : $();
+        let $levelPriceLabel = $();
```

Comment:
Avoid using let. If you want to make this cleaner looking than the ternary available before, do an IIFE but one layer higher (in other words, include the ` itemIsIdentifiedOrUserIsGM && item.system.stackGroup !== "coins"` as part of the inner function.